app/assets/stylesheets/card-footer.scss

```diff
@@ -0,0 +1,30 @@
+.card-footer {
+  background-color: $background-grey;
+  border-top: 1px solid $border-medium-grey;
+  bottom: 0;
+  font-size: $font-size-small;
+  left: 0;
+  line-height: $line-height-computed;
+  margin-left: 0;
+  min-height: $line-height-computed + 1;
+  position: absolute;
+  width: 100%;
+
+  .footer-container {
+    padding: 1px 5px;
```

Comment:
this seems to be from an outdated diff. I removed the wrong font-size line here a few minutes ago. Pronto doesn't say anything locally.